letsencrypt/cli.py

```diff
@@ -301,6 +301,12 @@ def _auth_from_domains(le_client, config, domains, plugins):
             raise errors.Error("Certificate could not be obtained")
 
     _report_new_cert(lineage.cert)
+    reporter_util = zope.component.getUtility(interfaces.IReporter)
+    reporter_util.add_message(
+        "Your certificate will expire at {0}. To obtain a new version of the "
+        "certificate in the future, simply run this client again.".format(
+            lineage.notafter().ctime()),
```

Comment:
FTR I'm getting "Your certificate will expire at Wed Jan 13 04:12:00 2016. To obtain a new version of the certificate in the future, simply run this client again."